package.json

```diff
@@ -116,10 +116,11 @@
     "sourceCache": true
   },
   "scripts": {
-    "build-dev": "rollup -c --environment BUILD:dev && build/run-tap --no-coverage test/build/dev.test.js",
+    "build-dev": "rollup -c --environment BUILD:dev",
     "watch-dev": "rollup -c --environment BUILD:dev --watch",
-    "build-min": "rollup -c --environment BUILD:production && build/run-tap --no-coverage test/build/min.test.js",
+    "build-min": "rollup -c --environment BUILD:production",
     "build-css": "postcss -o dist/mapbox-gl.css src/css/mapbox-gl.css",
+    "build-style-spec": "cd src/style-spec && npm run build && cd ../.. && cp -R src/style-spec/dist dist/style-spec",
```

Comment:
Any particular reason for `npm` rather than `yarn`?

Reply:
Nope, just habit, will change.

Reply:
Reverted to `npm` due to https://github.com/mapbox/mbgl-ci-images/issues/17